hayk.poghosyan/repo/Lesson_3/exercise_17/exercise_17.py

```diff
@@ -0,0 +1,24 @@
+# 17
+num_1 = int(input("Enter num1: "))
```

Comment:
RW: All the user input should be in one line.

Note. You should not use the techniques that you didn't pass in the Lesson 3